### PR TITLE
Displaying application versionName in general settings tab under "about" title. Fixed hebrew string for settings.

### PR DESCRIPTION
**Pull request by BarAgam**
Displaying application versionName in general settings tab under "about" title.
Fixed hebrew string for settings.

### CODE REVIEW

File: app/src/main/java/com/teamagam/gimelgimel/app/settings/SettingsActivity.java [hunk 1]
```diff
@@ -133,6 +134,9 @@ public void onCreate(Bundle savedInstanceState) {
       super.onCreate(savedInstanceState);
       addPreferencesFromResource(R.xml.pref_general);
       setHasOptionsMenu(true);
+      String aboutSummary =
```
Comment (teamagam5) [inline app/src/main/java/com/teamagam/gimelgimel/app/settings/SettingsActivity.java]: Hmmm... extract method?

File: app/src/main/res/values-iw/strings.xml [hunk 2]
```diff
@@ -77,4 +76,6 @@
   <string name="message_notification_summary">הודעה</string>
   <string name="message_notification_text">הודעה נכנסת חדשה</string>
   <string name="message_notification_title">הודעה חדשה</string>
+  <string name="pref_general_summary_template">גרסא %1$s</string>
+  <string name="pref_general_about_title">פרטים</string>
```
Comment (teamagam5) [inline app/src/main/res/values-iw/strings.xml]: אודות?

**Review by teamagam5 — APPROVED** [review]

File: app/src/main/res/values-iw/strings.xml (hunk 2 above)
Comment (BarAgam) [reply app/src/main/res/values-iw/strings.xml]: מה אתה בא לי בעברית

File: app/src/main/java/com/teamagam/gimelgimel/app/settings/SettingsActivity.java (hunk 1 above)
Comment (BarAgam) [reply app/src/main/java/com/teamagam/gimelgimel/app/settings/SettingsActivity.java]: K

**Pull request merged**